### DISCUSSION TITLE
피움 App QA 모음

### DISCUSSION THREAD

**Discussion by Choi-JJunho** [woowacourse-teams/2023-pium#474] [개발 논의]
# 서론
구글 플레이스토어에 출시된 피움 앱을 사용하며 발생하는 불편한 점을 모아둡니다.

# 플레이스토어
플레이스토어 링크
- https://play.google.com/store/apps/details?id=life.pium.twa

**Comment by Choi-JJunho** [comment]
- 앱에서 주소창이 보인다. 주소창이 안보이게 해줬으면 좋겠다.
<img src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/207ed452-1bb6-422d-9a44-83104b70d967" width="400px">

**Comment by Choi-JJunho** [comment]
- 앱 이름 `pium` -> `피움`
<img width="200px" alt="image" src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/3f077284-0358-4d7d-a003-1372a6731a51">

**Comment by Choi-JJunho** [comment]
- 식물 등록 문의하기 창에서 동일 이미지 여러번 업로드 시 취소됨(?)

<img src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/f193caa0-dca7-421f-bccc-267d93a479e3" width="400px">

**Comment by Choi-JJunho** [comment]
- 카카오 로그인 -> 뒤로가기 시 0.1초정도 보이는 로딩스피너 깜빡임 불편함

<img src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/c779dee2-46b9-47f3-8242-c10619affabc" width="400px">

**Comment by Choi-JJunho** [comment]
- 문의하기 - 구글 드라이브 뜸 - 뒤로 가기 시 에러 발생

**Comment by Choi-JJunho** [comment]
- 카카오 로그인을 누르고 카카오 홈까지 가버릴 수 있음 (앱에서 타사 웹으로 접근 가능)

<img src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/44e68614-adac-4051-855d-302d6411fd28" width="400px">

**Comment by Choi-JJunho** [comment]
- 이모지 입력 시 마지막에 `?` 들어가는 문제

<img src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/50b6918c-abc1-4188-a0e8-1211919b8914" width="400px">

**Comment by hozzijeong** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋ 문제가 많네요~ ㅋㅋㅋㅋㅋ

**Reply by WaiNaat** [reply]
초비상!!

**Reply by bassyu** [reply]
😿

**Comment by Choi-JJunho** [comment]
- 물주기 주기 입력 시 숫자 키패드가 뜨면 좋겠다

<img src="https://github.com/woowacourse-teams/2023-pium/assets/49794401/6b445f0b-6cd1-4517-ae6a-d6d95f8c996f" width="400px">
